src/pages/coupon-fraud/index.tsx

```diff
@@ -1,16 +1,60 @@
 import { useVisitorData } from '../../client/use-visitor-data';
 import { UseCaseWrapper } from '../../client/components/common/UseCaseWrapper/UseCaseWrapper';
-import { useCallback, useEffect, useState } from 'react';
-import FormControl from '@mui/material/FormControl';
-import TextField from '@mui/material/TextField';
-import Button from '@mui/material/Button';
-import Alert from '@mui/material/Alert';
+import { FunctionComponent, useCallback, useEffect, useState } from 'react';
+
 import Image from 'next/image';
-import { Stack, Typography } from '@mui/material';
 import { useRequestCouponClaim } from '../../client/api/coupon-fraud/use-coupon-claim';
 import React from 'react';
 import { USE_CASES } from '../../client/components/common/content';
 import { CustomPageProps } from '../_app';
+import styles from './couponFraud.module.scss';
+import formStyles from '../../styles/forms.module.scss';
+import classNames from 'classnames';
+import AirMax from './shoeAirMax.svg';
+import AllStar from './shoeAllStar.svg';
+import Plus from './buttonPlus.svg';
+import Minus from './buttonMinus.svg';
+import Alert from '../../client/components/common/Alert/Alert';
+import Button from '../../client/components/common/Button';
+
+const AIRMAX_PRICE = 356.02;
+const ALLSTAR_PRICE = 102.5;
+const TAXES = 12;
+
+const format$ = (price: number) => {
+  return new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+  }).format(price);
+};
+
+type ShoeProps = {
+  name: string;
+  color: string;
+  price: number;
+  image: any;
+  count: number;
+};
+
+const Shoe: FunctionComponent<ShoeProps> = ({ image, name, color, price, count }) => {
+  return (
+    <div className={styles.shoe}>
+      <Image src={image} alt="shoe" width={92} height={92} />
+      <div className={styles.shoeDescription}>
+        <div className={styles.shoeName}>{name}</div>
+        <div className={styles.shoeColor}>{color}</div>
+        <div className={styles.priceAndCount}>
+          <div className={styles.price}>{format$(price)}</div>
+          <div className={styles.count}>
+            <Image src={Minus} alt="Decrease item count" />
```

Comment:
Fake buttons insult me =)

Reply:
Yeah, I wanted to keep it MVP, but for your pleasure, I have sacrificed 13 minutes to make them work: https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/81/commits/5f6d2640ea2aa26f3315ba161595d0d196220ee9